.github/workflows/build-deploy-pudl.yml

```diff
@@ -33,29 +33,46 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: ${{ env.BUILD_REF }}
+          fetch-depth: 0
```

Comment:
I set this to 0 to ensure access to all the tags & branches so we can check whether there are any changes necessitating a build.

Reply:
Perfect! And it probably means we don't have to do as many gymnastics to get the git stuff working, either!